pkg/api/testing/serialization_test.go

```diff
@@ -286,7 +286,7 @@ func TestUnversionedTypes(t *testing.T) {
 		}
 		// Make sure it decodes correctly
 		if !reflect.DeepEqual(obj, versionDecodedObject) {
-			t.Errorf("%v: expected %#v, got %#v", obj, obj, versionDecodedObject)
+			t.Errorf("expected %#v, got %#v", obj, versionDecodedObject)
```

Comment:
This line and above two `Errorf()` lines follow the same error message style: "test obj: error message". 
Looks keeping them consistent is better?

It’s about aesthetics, just my suggestion.
Thanks.